src/extension_consts.ml

```diff
@@ -26,6 +28,10 @@ module Commands = struct
   let open_switches_documentation = ocaml_prefixed "open-switches-documentation"
 
   let open_sandbox_documentation = ocaml_prefixed "open-sandbox-documentation"
+
+  module Dune = struct
+    let open_current_dune_file = dune_prefixed "current-dune-file"
+  end
```

Comment:
This code still has to be updated to accommodate the change in command ID.

Reply:
Fixed, thanks!